lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/passport/issuecredential/IssueCredentialHandler.java

```diff
@@ -72,15 +97,44 @@ public APIGatewayProxyResponseEvent handleRequest(
             PassportCheckDao passportCheck =
                     dcsPassportCheckService.getDcsPassportCheck(resourceId);
 
-            PassportCriResponse passportCriResponse =
-                    PassportCriResponse.fromPassportCheckDao(passportCheck);
+            VerifiableCredential verifiableCredential =
+                    VerifiableCredential.fromPassportCheckDao(passportCheck);
 
-            return ApiGatewayResponseGenerator.proxyJsonResponse(
-                    HttpStatus.SC_OK, passportCriResponse);
+            SignedJWT signedJWT = generateAndSignVerifiableCredentialJwt(verifiableCredential);
+
+            return ApiGatewayResponseGenerator.proxyJoseResponse(
+                    HttpStatus.SC_OK, signedJWT.serialize());
         } catch (ParseException e) {
             LOGGER.error("Failed to parse access token");
             return ApiGatewayResponseGenerator.proxyJsonResponse(
                     e.getErrorObject().getHTTPStatusCode(), e.getErrorObject().toJSONObject());
+        } catch (JOSEException e) {
+            LOGGER.error("Failed to create VerifiableIdentityCredential");
+            return ApiGatewayResponseGenerator.proxyJsonResponse(
+                    HttpStatus.SC_INTERNAL_SERVER_ERROR, e.getLocalizedMessage());
         }
     }
+
+    private SignedJWT generateAndSignVerifiableCredentialJwt(
+            VerifiableCredential verifiableCredential) throws JOSEException {
+        Instant now = Instant.now();
+
+        // TODO: Set Subject and Issuer
+        JWTClaimsSet claimsSet =
+                new JWTClaimsSet.Builder()
+                        .claim(SUBJECT, "Subject")
+                        .claim(ISSUER, "Issuer")
+                        .claim(NOT_BEFORE, now.getEpochSecond())
+                        .claim(EXPIRATION_TIME, now.plusSeconds(600).getEpochSecond())
```

Comment:
The expiration time should probably be a config value grabbed from the param store.